gatsby/src/components/DetailsPageLayout/DetailsPageLayout.tsx

```diff
@@ -98,9 +114,30 @@ export const DetailsPageLayout: React.FC<DetailsPageLayoutProps> = ({
 
 			<Grid gutter="loose" data-testid="body">
 				{Menu && (
-					<GridItem cols={12} md={4} lg={3} className="hide-print">
-						<Menu />
-					</GridItem>
+					<>
+						<GridItem cols={12} md={4} lg={3} className="hide-print">
+							{isClient ? (
```

Comment:
I think if we move the logic and styles into the menu component, then it'll be easier to replace all instances with the nds stacked-nav directly if/when we move the logic there.